### DISCUSSION TITLE
Downloading and caching globally

### DISCUSSION THREAD

**Discussion by jodersky** [com-lihaoyi/mill#4070] [Q&A]
I'm looking into downloading a static binary as part of a mill task. This binary will be used over and over again, and I'd like to cache the file machine-wide (basically share it even across completely different mill projects).

Is there already something in mill which provides this out of the box? I could probably do this manually and just dump the file to a hard-coded location such as `~/.cache/???`, but a built-in feature would be more convenient.

**Comment by alexarchambault** [comment]
#4066 shows how to use coursier for that. For a broader discussion about that problem, see #3930 though.
